integrations/facebook-pixel/test/index.test.js

```diff
@@ -1344,6 +1343,38 @@ describe('Facebook Pixel', function() {
         );
         assertEventId(window.fbq);
       });
+
+      it('should send facebook pixel properties for product viewed', function() {
+        analytics.track('Product Viewed', {
```

Comment:
"Product Viewed" events are formatted a little differently than what you have here: https://segment.com/docs/spec/ecommerce/v2/#product-viewed

Reply:
Got that.